lib/exceptions.json

```diff
@@ -1,2 +1,12 @@
 {
+ "webrtc": [
+    {
+      "rule": "headers.copyright.no-match"
```

Comment:
The name of the rule is `headers.copyright`. The current exception system is not granular enough to skip specific errors raised by a rule.